docs/user/logs.md

```diff
@@ -0,0 +1,597 @@
+  },
+  "body": "{\"level\": \"warn\",\"message\": \"This is the actual message\",\"tenant\": \"myTenant\",\"trace_id\": \"123\"}"
+}
+```
+
+Hereby, all information identifying the actual source of the log like the Container, Pod and Namespace name are getting enriched as resource attributes following the [Kubernetes conventions](https://opentelemetry.io/docs/specs/semconv/resource/k8s/). Further metadata like the original file name and channel are enriched as log attributes following the [log attribute conventions](https://opentelemetry.io/docs/specs/semconv/general/logs/). The `time` value provided in the container runtime log entry is used as `time` attribute in the new OTEL record, as it is very close to the actual time when the log happened. Additionally, the `observedTime` is set with the time when the agent actual read the log record as recommended by the [OTEL log specification](https://opentelemetry.io/docs/specs/otel/logs/data-model/#field-observedtimestamp). The actual log payload gets moved into the OTLP `body` field.
+
+#### JSON Parsing
+
+If the value of the `body` is a JSON document, the value gets parsed and all JSON root attributes will get enriched as additional log attributes. The orginial body gets moved into the `log.original` attribute (managed via LogPipeline `input.application.keepOriginalBody: true` attribute).
+
+The resulting OTLP record will look like this:
+
+```json
+{
+  "time": "2022-05-23T15:04:52.100000000Z",
+  "observedTime": "2022-05-23T15:04:52.200000000",
+  "attributes": {
+   "log.file.path": "/var/log/pods/myNamespace_myPod-<containerID>/myContainer/<containerRestarts>.log",
+   "log.iostream": "stdout",
+   "log.original": "{\"level\": \"warn\",\"message\": \"This is the actual message\",\"tenant\": \"myTenant\",\"trace_id\": \"123\"}",
+   "level": "warn",
+   "tenant": "myTenant",
+   "trace_id": "123",
+   "message": "This is the actual message"
+  },
+  "resourceAttributes": {
+    "k8s.container.name": "myContainer",
+    "k8s.container.restart_count": "<containerRestarts>",
+    "k8s.pod.name": "myPod",
+    "k8s.namespace.name": "myNamespace"
+  },
+  "body": ""
+}
+```
+
+#### Severity Parsing
+
+As typicaly a log message has a log level written to a field `level`, the agent will try to parse the log attribute `level` with a [severity parser](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/pkg/stanza/docs/operators/severity_parser.md). If that is successful, the log attribute gets transformed into the OTEL `severityText`and `severityNumber` attributes.
+
+#### Trace Parsing
+
+OTLP natively supports attaching trace context to log records. Hereby, the ganet will try to parse it from log attributes with name `trace_id`, `span_id` and `trace_flags`.
+
+#### Log Body Determination
+
+As the actual log message should be located in the `body` attribute, the agent will move a log attribute with name `message` or `msg` into the `body`.
```

Comment:
```suggestion
Because the actual log message is typically located in the `body` attribute, the agent moves a log attribute called `message` (or `msg`) into the `body`.
```
why "should be" - in which cases is that typically not the case? I assume those would be rare edge cases?